conda-forge.yml

```diff
@@ -5,8 +5,7 @@ build_platform:
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
-# Workaround for https://github.com/conda/conda-libmamba-solver/issues/446
-conda_build_tool: conda-build+classic
+conda_build_tool: mambabuild
```

Comment:
I recommend to delete that line entirely.

Reply:
But then we run into https://github.com/conda/conda-libmamba-solver/issues/446